sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ServiceBusReceiveLinkProcessorTest.java

```diff
@@ -573,7 +576,7 @@ void receivesFromFirstLink() {
         assertFalse(processor.hasError());
         assertNull(processor.getError());
 
-        verify(link1).addCredits(eq(PREFETCH));
+        verify(link1, times(3)).addCredits(eq(PREFETCH));
```

Comment:
Can you add a reason why is times(3) and all the other instances? I don't think I'll remember why this is the case.. there are messages+1 in some cases.